apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/Numeric.cs

```diff
@@ -388,6 +424,21 @@ public override string ToString()
             int originalIntegerPlaces = Math.Max(integerText.Length - signLength - Type.Scale, 0);
             int resultIntegerPlaces = Math.Max(originalIntegerPlaces, 1);
 
+            // How many places we have leftover. We may have negative places leftover.
```

Comment:
I don't think this comment describes the value of this variable. Or at least, I don't understand it if it does.

Perhaps rename to `availableDecimalPrecision`, as basically "the maximum number of decimal places we're allowed to include in the result, based on how many digits are used to represent the integer part"? (I think that's what it means - and if not, again, I'm confused.)